editor/code_editor.h

```diff
@@ -244,13 +244,14 @@ class CodeTextEditor : public VBoxContainer {
 	void update_editor_settings();
 	void set_error(const String &p_error);
 	void set_error_pos(int p_line, int p_column);
+	void get_error_pos(int &p_line, int &p_column);
```

Comment:
Can use a `Pointi` here rather then return args.

Reply:
Yeah, and `p_line` is an incorrect name, if the variable is intended to be returned it should be named as `r_line`.